com.microsoft.mrtk.standardassets/Materials/Backplate/CanvasBackplate_SeeItSayIt.mat

```diff
@@ -0,0 +1,82 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
```

Comment:
Why does the Canvas see-it-say-it require a bespoke material? Is it not the same material as the rest of the backplates?

Reply:
Removed unnecessary material and reassigned CanvasBackplate material.